src/core_ocean/shared/mpas_ocn_mesh.F

```diff
@@ -116,6 +116,9 @@ module ocn_mesh
       meshDensity, &! density of mesh
       angleEdge        ! angle the edge normal makes with local east
 
+   real(kind=RKIND), public, dimension(:), allocatable :: &
+      invAreaCell   ! inverse of area of each cell
```

Comment:
I added an array for `invAreaCell` to pre-calculate this value since it is used throughout the model.  I have yet to propagate this to any routines aside from diagnostics.